### PR TITLE
Eahw2815/timecard md

### PR DESCRIPTION
**Pull request by Xeena-Newman**

### CODE REVIEW

File: src/documents/images/Screenshot 2023-06-22 at 08.15.32.png
Comment (IanFisherHO) [inline src/documents/images/Screenshot 2023-06-22 at 08.15.32.png:1]: Can you rename the screenshot files to have more meaningful names please?

File: src/documents/timecard.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+The timecard page enables the user to record their work times via a sequence of screens. Currently the only developed timecardPeriodType is Shift. After choosing this radio button option, the user is taken to the TimecardEntriesList component where they can input the times and dates worked:
+
+![Alt text](images/Screenshot%202023-06-22%20at%2008.15.32.png)
+
+
+After clicking save, the screen displays the recorded shift with the option to remove or edit it: 
+
+![Alt text](images/Screenshot%202023-06-22%20at%2008.46.42.png)
+
+When editing the saved time, the function 'isFinishTimeOnNextDay' calculates whether the time is on the next day and the UI notifies the user with dynamic text:
```
Comment (IanFisherHO) [inline src/documents/timecard.md:13]: Maybe reword 'isFinishTimeOnNextDay' functionality since it works on insert as well as edit.
The main point to get across is the reason why it's there, so users on night shift can come on and quickly type in  their hours without having to manually set the dates.

File: src/documents/timecard.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+![Alt text](images/Screenshot%202023-06-22%20at%2009.26.14.png)
+
+The date input is conditionally rendered by clicking the checkbox. The dates are automatically inserted. If 'isFinishTimeOnNextDay' is true, the finish date is set to the next day. These can be manually changed: 
+
+![Alt text](images/Screenshot%202023-06-22%20at%2010.14.50.png)
+
+The user is returned to the the Timecard of the start date:
```
Comment (IanFisherHO) [inline src/documents/timecard.md:21]: The user is never re-routed, they will remain on the day they were originally viewing.

File: src/documents/timecard.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Validation is in place to prevent invalid time iputs: 
+
+![Alt text](images/Screenshot%202023-06-22%20at%2011.19.05.png)
+
+
+A banner notifying the user that their hours have been changed is displayed only if the day is edited to move from the current timecard date:
+
+![Alt text](images/Screenshot%202023-06-22%20at%2011.51.30.png)
+
+Finish time can be left blank without error, the system will asume the finish date is the same day:
```
Comment (IanFisherHO) [inline src/documents/timecard.md:40]: The system will assume it's an open ended shift until the user adds the time at a later date. The date field may default to the timecard date but without the time it's not set.

**Review by IanFisherHO — CHANGES_REQUESTED** [review]

File: src/documents/timecard.md [hunk 4]
```diff
@@ -10,15 +10,15 @@ After clicking save, the screen displays the recorded shift with the option to r
 
 ![Alt text](images/Screenshot%202023-06-22%20at%2008.46.42.png)
```
Comment (IanFisherHO) [inline src/documents/timecard.md:11]: The links will need updating to match the new file names.

File: src/documents/timecard.md [hunk 5]
```diff
@@ -10,15 +10,15 @@ After clicking save, the screen displays the recorded shift with the option to r
...
+The user is notified if the finish time is on the next day using dynamic next:
 
 ![Alt text](images/Screenshot%202023-06-22%20at%2009.26.14.png)
 
 The date input is conditionally rendered by clicking the checkbox. The dates are automatically inserted. If 'isFinishTimeOnNextDay' is true, the finish date is set to the next day. These can be manually changed: 
 
 ![Alt text](images/Screenshot%202023-06-22%20at%2010.14.50.png)
 
-The user is returned to the the Timecard of the start date:
+The user remains on the start date of the Timecard:
```
Comment (IanFisherHO) [inline src/documents/timecard.md:21]: The user remains day they were currently viewing, referencing the start date is misleading

**Review by IanFisherHO — DISMISSED** [review]

**Review by sewerynmi — APPROVED** [review]

**Review by JWeston-HO — APPROVED** [review]
